README.md

```diff
@@ -37,6 +37,14 @@ export AWS_SESSION_TOKEN=<SESSION_TOKEN>  # Optional
 # Use --help for details.
 ```
 
-> **Warning**
-> We do not support ddl replication yet. Any ddl operation will cause the incremental replication stop. You need to manually run the DDL on target table and then restart the incremental replication.
-> Restart incremental replication without manually runing DDL on target table may cause data loss.
+## Supported DDL Operations
+
+* Drop column
+* Alter (rename) column
+* Alter column' (default) value
+* Drop/truncate tables
+* Change column type
+* Multi schema change
+* Add table partition
```

Comment:
```suggestion
- Add column
- Drop column
- Alter column rename
- Alter column modify column type
- Drop table
- Truncate table
```

> **Note**
> 1. Snowflake does not support partition table, tidb2dw will view table with multiple partitions as ordinary table.
> 2. Snowflake does not support update column default value, be careful.
> 3. The type mapping from TiDB to Snowflake is defined [here](./snowsql/types.go)

Reply:
Snowflake has a lot of limitations on modifying column type, should be careful. refer to https://docs.snowflake.com/en/sql-reference/sql/alter-table-column

Reply:
4. Should execute any DML before DDL or will report error.

Reply:
## Supported DDL Operations

All DDL which will change the schema of table are supported (except index related), including:

- Add column
- Drop column
- Rename column
- Modify column type
- Drop table
- Truncate table

> **Note**
> 1. Snowflake does not support partition table, tidb2dw will view table with multiple partitions as ordinary table.
> 2. Snowflake has a lot of limitations on modifying column type, like Snowflake does not support update column default value, refer to [Snowflake Docs](https://docs.snowflake.com/en/sql-reference/sql/alter-table-column).
> 3. The type mapping from TiDB to Snowflake is defined [here](./snowsql/types.go).
> 4. Should execute any DML before DDL or will report error.